recipes/runstats/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "runstats" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/grantjenks/python-runstats/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 48a0cd627ee5e854990960ca8920a789922fe7477d0c6d469f9f5baf6e007294
+
+build:
+  noarch: python
```

Comment:
If you need a compiler, it cannot be `noarch`

Reply:
Thanks for the review and good catch. I started from `grayskull pypi` which introduced the `noarch` (was not aware when starting that it uses `cython`).

The linter now says: `Non noarch packages should have python requirement without any version constraints` which is why I removed the python constrained in `host` and `run` but that does not feel correct to me.